docs/user_manual/type_data.md

```diff
@@ -0,0 +1,514 @@
+database. As such, they must be performed by a MISO adminstrator.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Designs" %}
+
+Note: This item only applies if your site uses [detailed sample mode](site_configuration.html#detailed_sample_mode).
+
+A library design is a grouping of [selection](#library_selection_types) and [strategy](#library_strategy_types) types.
+The library designs that are available for a library depend on the sample class of the aliquot that the library was
+propagated from. Modifications to library designs can only be made via direct access to the MISO database. As such,
+they must be performed by a MISO adminstrator.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Selection Types" %}
+
+Library selection type is defined by [NCBI](https://www.ncbi.nlm.nih.gov/) as the "method used to enrich the target in
+the sequence library preparation." This property is required in order to submit libraries to the SRA (see the
+[SRA section](sra.html)). The values available in MISO should be limited to the default options, which are the same as
+defined in the SRA schema.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Library Strategy Types" %}
+
+Library strategy type is defined by [NCBI](https://www.ncbi.nlm.nih.gov/) as the "sequencing type intended for this
+library." This property is required in order to submit libraries to the SRA (see the [SRA section](sra.html)). The
+values available in MISO should be limited to the default options, which are the same as defined in the SRA schema.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Kit Descriptors" %}
+
+A kit descriptor describes a kit product that is used for library or run preparation. The types of kits available in
+MISO are:
+
+* Library kit - used for libraries
+* Sequencing kit - not currently used in MISO
+* Clustering kit - used for sequencing containers
+* Multiplexing kit - used for sequencing containers
+* Extraction kit - not currently used in MISO
+
+Modifications to kit descriptors can only be made via direct access to the MISO database. As such, they must be
+performed by a MISO adminstrator.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Indices" %}
+
+Indices, also known as barcodes or primers, are sequences that are added to libraries in order to identify which
+library each individual DNA fragment came from in a multiplexed pool. An index family is a collection of these indices
```

Comment:
Single-plexed pools can also have indices (and MISO warns when they don't).

Reply:
True... But do they really have a purpose in single-plexed pools? Is there reason to change the wording here?

Reply:
They can also be used to confirm no sample swaps occurred (confirming that the library which was sequenced has the same index as the library in the single-plex pool in MISO). I don't feel strongly about changing the wording.